### PR TITLE
Adds the super heavy pistol

### PR DESCRIPTION
**Pull request by KingOfThePing**
This PR adds the Kumar Arms 2557, a super heavy pistol. 

IC description:

The Kumar Arms 2557 is a newly designed type of "super-heavy" pistol.  It features a light-weight polymer pistol grip, a bulky plasteel frame and an extra long barrel. It is chambered in the equivalent of the .577 Nitro Express rifle cartridge, and with newly developed Zavodskoi propellant, the length of the casing could be reduced significantely. Kumar Arms guarantees your enemy's armor penetrated or your money back. It features a revolutionary automatic revolving bolt system, which ensures accuracy at the cost of fire rate.

The 2557 is intended to be a very hard hitting, slow firing pistol. This PR includes it's sprites and custom sounds. Using this pistol without wielding it will break your hand and fly out of your hand, except for Unathi and G2 (or any larger mobs).

It will be available on the Uplink, for away site loot tables and in general as an addition for the palette of Zavodskoi firearms.

The magazine holds 5 rounds. The damage values are not final.

Right now the sounds are missing since I am still working on them, one magazine icon is also not updating properly so far, this is why this PR is still a WIP.

### CODE REVIEW

**Comment by KingOfThePing** [comment]
!wip

**Comment by FayeLovesU** [comment]
Maybe non-unathi / g2 can shoot it one-handed if they're in a RIGsuit?

**Review by TheDanseMacabre — CHANGES_REQUESTED** [review]
Not personally a fan that the sprite is a reference to a gun from Cyberpunk, even if the spritework is good. More importantly, however, I think if you're going to add a new cartridge to the game, it should be an original cartridge. .577 Nitro Express was designed in the 1890s for hunting elephants. How about ".590 Kumar"?

**Comment by KingOfThePing** [comment]
> Not personally a fan that the sprite is a reference to a gun from Cyberpunk, even if the spritework is good. More importantly, however, I think if you're going to add a new cartridge to the game, it should be an original cartridge. .577 Nitro Express was designed in the 1890s for hunting elephants. How about ".590 Kumar"?

I respect but dont agree and thus discard your opinion about references being inheretly bad/your quest to purge them. 

Regarding the caliber I gave it a thought this morning and I think you are right. I even wrote it is the EQUIVALENT of .577 Nitro Express. So I think some in-universe sci-fi bullshit caliber would fit better, as you said. I will think about what exactly to change it to and will replace the existing caliber with a more lore-friendly one.

**Comment by KingOfThePing** [comment]
The gun is now chambered in .599 Kumar Nitro (probably a working title)

**Comment by KingOfThePing** [comment]
I decided on .599 Kumar Super as the bullshit sci-fi caliber.

Sounds have also been added. All CC0.

**Comment by KingOfThePing** [comment]
!review

**Comment by KingOfThePing** [comment]
wtf is this bots problem

**Comment by KingOfThePing** [comment]
I would add one to the armoury if I didnt knew Matt would kill me

**Comment by Arrow768** [comment]
Will be evaluated together with #13772 and the other changes in that PR

**Comment by KingOfThePing** [comment]
I am not dead set on these damage values. I think the pistol is strong, yes, but I balanced it out with a 5 shot magazine and a fire_delay of 18.

File: code/modules/projectiles/projectile/bullets.dm [hunk 1]
```diff
@@ -232,6 +232,12 @@
 	maim_type = DROPLIMB_BLUNT
 	anti_materiel_potential = 2
 
+/obj/item/projectile/rifle/kumar_super
+	damage = 35
+	armor_penetration = 20
+	weaken = 2
```
Comment (NonQueueingMatt) [inline code/modules/projectiles/projectile/bullets.dm:238]: Get rid of this as a stun is a bit too powerful. Otherwise, looks fine, I guess?

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

File: code/modules/projectiles/projectile/bullets.dm (hunk 1 above)
Comment (KingOfThePing) [reply code/modules/projectiles/projectile/bullets.dm:238]: As you wish

**Review by NonQueueingMatt — APPROVED** [review]

**Review by JohnWildkins — APPROVED** [review]

**Comment by Zelmana** [comment]
@KingOfThePing 

I am not sure if this is because its from your fork or what- but I can't see the icons well, or don't know how to access them.

This caught my eye as it's described as having a "revolutionary automatic revolving bolt system". Is this a revolver weapon?

If so, why does it have a magazine?
Additionally, the bolt system revolves? I am not sure what this is a reference to mechanically. The bolt carrier group revolves or is turned? Thinking about this I am not sure what this is.

**Comment by KingOfThePing** [comment]
@Zelmana It's literally a sci fi bullshit pistol. I needed a justification on a) why it fires so slow (muh bolt rotates) and b) still be able to call it a pistol, so it has a magazine. I attach the sprites here, so you can look at them.

Also, as silly and stupid as they are, magazine fed revovlers exis

t

**Pull request merged**